examples/graphics_settings.rs

```diff
@@ -203,9 +214,6 @@ struct Opt {
     /// what level of MSAA to try to use (1, 2, 4, 8)
```

Comment:
Maybe we should change the comment here (and the one in line 241) to only mention levels 1 and 4 for the time being?
Or do we leave it be?